### PR TITLE
Refresh the project - Node > 10 | Ember > 3.16 | No mixins

### PR DESCRIPTION
**Pull request by Exelord**
- Removed mixins
- Support for Ember >3.16
- Support for Node >10

### CODE REVIEW

File: addon/components/image/template.hbs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<img ...attributes
+  class={{this.class}}
+  width={{this.width}}
+  height={{this.height}}
+  src={{this.src}}
+  alt={{this.alt}}
+  title={{this.title}}
+  onError={{this.onError}}>
```
Comment (herzzanu) [inline addon/components/image/template.hbs]: new line :)

File: tests/dummy/app/components/avatar-initials/template.hbs [hunk 2]
```diff
@@ -3,17 +3,17 @@
...
-          seedText=seedText
-          defaultName=defaultName
-          textColor=textColor
-          fontSize=fontSize
-          fontWeight=fontWeight
-          fontFamily=fontFamily
-          image=image
-        }}
+        <EmberInitials class="img-circle"
+          @size={{size}}
```
Comment (herzzanu) [inline tests/dummy/app/components/avatar-initials/template.hbs]: Use of `this.` for all the values? So `@size={{this.size}}` etc

File: addon/components/image/index.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import Component from '@ember/component';
```
Comment (herzzanu) [inline addon/components/image/index.js]: `import Component from '@glimmer/component';`

File: addon/components/image/template.hbs (hunk 1 above)
Comment (Exelord) [reply addon/components/image/template.hbs]: in hbs? :)

File: tests/dummy/app/components/avatar-initials/template.hbs (hunk 2 above)
Comment (Exelord) [reply tests/dummy/app/components/avatar-initials/template.hbs]: Demos will be rewritten in the future :)

File: addon/components/image/template.hbs (hunk 1 above)
Comment (herzzanu) [reply addon/components/image/template.hbs]: Disregard, not sure what's with this comment 😄

File: addon/components/image/index.js (hunk 3 above)
Comment (Exelord) [reply addon/components/image/index.js]: That wont be an option for now. We would need to rewrite all components to support `this.args` :/

**Pull request merged**